<launch>
    <!-- Load parameters from a YAML file -->
    <rosparam file="$(find pwm_driver)/config/config.yaml" command="load"/>

    <!-- Launch the pwm_driver_node -->
    <node name="pwm_driver_node" pkg="pwm_driver" type="pwm_driver_node" output="screen">
        <param name="pwm_frequency" value="50"/>
        <param name="pwm_ms_bias" value="0.0"/>
        <param name="thruster_num" value="8"/>
         <!-- Kill all signals safety timeout -->
        <param name="timeout_seconds" value="2.0"/> 
        <!-- Add other parameters as needed -->
    </node>
</launch>
